hibike/unit_tests/hibike_process_tests.py

```diff
@@ -0,0 +1,65 @@
+"""
+Unit tests for functions in hibike_process.
+"""
+import unittest
+import time
+import serial
+from spawn_virtual_devices import spawn_device, get_virtual_ports
+from hibike_process import identify_smart_sensors
+import hibike_message as hm
+
+class IdentifySmartSensorsTests(unittest.TestCase):
+    """
+    Tests for `identify_smart_sensors`.
+    """
+    VIRTUAL_DEVICE_TYPES = ["LimitSwitch", "YogiBear", "YogiBear",
```

Comment:
This is just an example of a set of sensors that might be put on a robot. If there's anything that might be factored out, it would be `run_with_random_data`